ministryofjustice/analytics-platform-control-panel
# Provide a prometheus metrics endpoint that can be consumed by the monitoring services

## What

This uses the [korfuri django-prometheus](https://github.com/korfuri/django-prometheus/) library to provide a prometheus endpoint.


## How to review

1. Checkout the repo
2. Build and run the repo locally to ensure you can see the localhost:8000/metrics endpoint (ensuring you don't try and add the trailing slash)
3. Check the number of django_control_panel_login events
4. Login again
5. Check the number has incremented.
6. Deploy to dev
7. Check to see after deploying to dev, that you can see the metrics endpoint
8. Ask @neillturner very nicely to see if his monitoring can consume the endpoint, and ensure that he has both data statistics and can see the django_control_panel_login events
9. Do a happy dance


## This 

Fixes the control panel side of [ANPL-701](https://dsdmoj.atlassian.net/browse/ANPL-701)
